core-java-modules/core-java-graphql/src/main/java/com/baeldung/graphql/Constants.java

```diff
@@ -0,0 +1,9 @@
+package com.baeldung.graphql;
+
+public class Constants {
+
+    public static final String QUERY_PARAMETER = "query";
+
+    private Constants() {}
```

Comment:
Let's remove this boilerplate. While we might try to appease code scanning tools (which would also want the class to be `final` in this case) for these articles, less is more.